### PR TITLE
Node.js 14 preview

### PR DESCRIPTION
**Pull request by mhoeger**
Resolves https://github.com/Azure/azure-functions-nodejs-worker/issues/327

(Note: node.js 14 support not added for functions v2, only v3)

### CODE REVIEW

**Comment by mhoeger** [comment]
Hey @anthonychu and @pragnagopa - mind reviewing? thanks!

**Review by pragnagopa — APPROVED** [review]
LGTM!

File: src/WorkerChannel.ts [hunk 1]
```diff
@@ -101,6 +101,13 @@ export class WorkerChannel implements IWorkerChannel {
     if (this._v1WorkerBehavior) {
       if (version.startsWith("v12.")) {
         systemWarn("The Node.js version you are using (" + version + ") is not fully supported with Azure Functions V2. We recommend using one the following major versions: 8, 10.");
+      } else if (version.startsWith("14.")) {
```
Comment (anthonychu) [inline src/WorkerChannel.ts]:
```suggestion
      } else if (version.startsWith("v14.")) {

```

image

**Review by anthonychu — CHANGES_REQUESTED** [review]
I think the `v14.` comment is the only blocker. Not blocking but curious about only checking for specific LTS versions to reject but we don't seem to do anything about non-LTS versions like 11 or 13.

**Pull request merged**